content/en/docs/concepts/policy/limit-range.md

```diff
@@ -1,6 +1,5 @@
 ---
 reviewers:
-- sftim
 title: Limit Ranges

```

Comment:
Additional nits, possible edits:
clean up all extra spaces
line 12: Limit Range is a policy to constrain Pod and Container resources in a namespace.
lines 21-24: end sentence with period for all?
line35: Add header?, `###Overview of Limit Range`
line 38: Add comma and use plural?, Users create resources like Pods, Containers, and PersistentVolumeClaims in the namespace.
lines 39-40, clarify, possible edit: The `LimitRanger` admission controller enforces default requests and limits for all Pods and Containers that do not set compute resource requirements, and 
tracks usage to ensure it does not exceed resource minimum, maximum, and ratio defined in any `LimitRange` present in the namespace
line 47: do you need to specify, `testing` in quotes. Could just say `a namespace`
lines 50-51, clean up extra space , first-come-first-served is hard to translate -- can you reword?, is it Pods or Containers OR Pods and Containers?
In the case where the total limits of the namespace is less than the sum of the limits of the Pods or Containers,
there may be contention for resources. The Pod or Container that requested the resource first is given higher priority?
line 55, possible edit, `## Limiting Container compute resources`
line 58: A Pod with a 4 containers
line 59: Remove?, the purpose of the example is to showcase how each container within the pod is handled by the LimitRanger admission controller.
line 61: spelling of kubectl, Create a namespace `limitrange-demo` using the following `kubectl` command
line76: you are referring to the `field` names in this description. Is this needed? Readers have the manifest.
`This object defines min and max Memory/CPU limits, default cpu/Memory requests and default limits for CPU/Memory resources to be apply to containers.`
line 78: Create the `limit-mem-cpu-per-container` `LimitRange`.
lines 107-108:`### Container spec with valid cpu and memory requests and limits`
Be consistent with the title for all container/resources,
   View the `busybox-cnt01` resource configuration:
line 127-130: check capitalization and punctuation at end of sentences
line 135: View the `busybox-cnt02` container configuration:
line 162: View the busybox-cnt03 container configuration:
line 216, `## Limiting Pod compute resources`
lines 217-218, reword/spelling or remove: The previous section highlights the usage of a LimitRange to limit resources at Container level, the following section will be focusing on leveraging LimitRange to constrain resources at Pod level.
line 263: add possible note above instead of mentioning here?: LimitRange validations occur at the Pod Admission stage, not on Running pods.
line 266: ## Limiting Storage resources
line 322-324: clean up and the field is lowercase, `maxLimitRequestRatio`?
If `LimitRangeItem.maxLimitRequestRatio` is specified in the `LimitRangeSpec`, the named resource must have a request and limit that are both non-zero where limit divided by request is less than or equal to the enumerated value.
Create the following `LimitRange` with the memory limit set at twice the amount of the memory request for any pod in the namespace.

line 355, `ADD SPACE after 3`: 
The pod creation failed as the ratio of `3` is greater than the enforced limit of `2`: